ci/docker/ubuntu-20.04-clang-cpp.dockerfile

```diff
@@ -0,0 +1 @@
+ubuntu-18.04-clang-cpp.dockerfile
```

Comment:
This seems not to be what it says on the label - are we working with 20.04, or 18.04?

Reply:
This is a symbolic link to 18.04. Both 18.04 and 20.04 are used for the containers due to the compilers that are available across the linux versions as packages